docs/guides/spring-security-docs-guides.gradle

```diff
@@ -28,7 +30,7 @@ ext.spec = copySpec {
 }
 
 def getDownloadUrl() {
```

Comment:
Is this still necessary?

Reply:
It isn't, I'm gonna delete it and the references. Thanks!